_includes/code_snippet.html

```diff
@@ -0,0 +1,17 @@
+{% assign code = include.code %}
+<style>
+.copyButton {
+	background: transparent;
+    border: none;
+}
+</style>
+<script src="https://cdn.jsdelivr.net/npm/clipboard@2.0.6/dist/clipboard.min.js"></script>
```

Comment:
This seems like a lot of code and pinned to a version that may become outdated. Could we use our own code here instead?

Reply:
Sure, I think `navigator.clipboard.writeText("{{code}}");` might just work.
 
It's working in a plain html page.

`bundle exec jekyll serve --incremental` takes really long to serve changes, any tips to speed that up?

Reply:
That was really an unnecessary library for this usecase.

A single line of custom code worked perfectly in Chrome, FF and Safari!

Nice catch! 😄 

```js
document.querySelector("#copyButton")
    .addEventListener("click", () => navigator.clipboard.writeText("{{code}}"));
```

Reply:
> `bundle exec jekyll serve --incremental` takes really long to serve changes, any tips to speed that up?

Could delete various files here.




> Sure, I think `navigator.clipboard.writeText("{{code}}");` might just work.

Sounds good 👍🏻

Reply:
Looks mostly good: https://developer.mozilla.org/en-US/docs/Web/API/Clipboard

Should be able to stop it breaking on IE by only showing the icon if `navigator.clipboard` is truthy.

Reply:
Are we supporting IE? 

I assumed the targeted users of brew won't be using IE

Reply:
Done.

```js
    if(!navigator.clipboard) {
        document.getElementById("copyButton").remove();
    }
```